src/llmcompressor/modifiers/utils/pytorch_helpers.py

```diff
@@ -9,27 +9,12 @@
 from llmcompressor.pytorch.utils import tensors_module_forward, tensors_to_device
 
 __all__ = [
-    "EarlyStopException",
     "apply_pad_mask_to_batch",
     "run_calibration_forward",
     "is_moe_model",
 ]
 
 
-class EarlyStopException(Exception):
-    """
```

Comment:
We had two EaryStopExceptions?

Reply:
Yep! They basically serve the same function, but in the new version, cpu offloading is handled by the `IntermediatesCache` more robustly